From patchwork Tue Nov 12 21:00:27 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 11240153
Return-Path: <SRS0=xhWs=ZE=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 307AC13B1
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 Nov 2019 21:00:47 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.kernel.org (Postfix) with ESMTP id 1030021A49
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue, 12 Nov 2019 21:00:47 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (2048-bit key) header.d=canb.auug.org.au header.i=@canb.auug.org.au
 header.b="MVzlhpxs"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1727069AbfKLVAo (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 12 Nov 2019 16:00:44 -0500
Received: from ozlabs.org ([203.11.71.1]:38799 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1726697AbfKLVAn (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 12 Nov 2019 16:00:43 -0500
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 47CKs43JPvz9sNH;
        Wed, 13 Nov 2019 08:00:40 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1573592440;
        bh=Fl+cnxImbUdxKiW7FdnzmPITLo/GSGUVAFEIaaL5eo0=;
        h=Date:From:To:Cc:Subject:From;
        b=MVzlhpxs4rSOX1QJUEOnQslPQrXJ1R279YUQO/nyPodUDCZYiQgnotS9Te1U5QlvE
         rkBkppTlI2//CHrmsMnZyhuchJkETC9pz1EtBF6vCJr09TOSpzKB3h9UCdb8rlAnZX
         8sGd9jeNw4qYY/99JoP4dz31tKgFJ6Rq5rfEJ1WVBLDVEpRnOHpQM4ZYBm4k8z1ezh
         Z6aMmYQEsScFndpTBk7mkS+sFoKFNQ99f+lWPkdPZnPj95A//O1LNW8MxMLH3eSAGZ
         h84WtvE3mGGKRePiV6gjsk5pDU+EN7qXwPbrn+q9ivxs5lBy89n14pEjVb86NM6/9U
         8q7daU8fIDbFQ==
Date: Wed, 13 Nov 2019 08:00:27 +1100
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>,
 Radim =?utf-8?b?S3LEjW3DocWZ?= <rkrcmar@redhat.com>,
 KVM <kvm@vger.kernel.org>
Cc: Linux Next Mailing List <linux-next@vger.kernel.org>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Junaid Shahid <junaids@google.com>,
        Thomas Gleixner <tglx@linutronix.de>
Subject: linux-next: manual merge of the kvm-fixes tree with Linus' tree
Message-ID: <20191113080027.7f97187e@canb.auug.org.au>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm-fixes tree got a conflict in:

  virt/kvm/kvm_main.c

between commit:

  1aa9b9572b10 ("kvm: x86: mmu: Recovery of shattered NX large pages")

from Linus' tree and commit:

  8a44119a98be ("KVM: Fix NULL-ptr deref after kvm_create_vm fails")

from the kvm-fixes tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc virt/kvm/kvm_main.c
index 4aab3547a165,0dac149ead16..000000000000
--- a/virt/kvm/kvm_main.c
